src/quick-lint-js/diag/diagnostic-types-2.h

```diff
@@ -3307,6 +3307,14 @@ struct Diag_Class_Async_On_Getter_Or_Setter {
   Source_Code_Span async_keyword;
   Source_Code_Span getter_setter_keyword;
 };
+
+struct Diag_Multiple_Export_Defaults {
+  [[qljs::diag("E0715", Diagnostic_Severity::error)]] //
+  [[qljs::message("cannot export default more than once", ARG(second_export_default))]]   //
+  [[qljs::message("export default previously appeared here", ARG(first_export_default))]] //
```

Comment:
Nit: Message here should match the title in docs/errors/E0715.md. (It doesn't have to be an exact match.)

For example, you could title the docs "cannot export default more than once" or you could change the first message to "cannot use multiple `export default` statements in one module".